modules/preview-react/menu/lib/Menu.tsx

```diff
@@ -1,12 +1,20 @@
 import * as React from 'react';
-import styled from '@emotion/styled';
 
 import {MenuItemProps} from './MenuItem';
 import {Card} from '@workday/canvas-kit-react/card';
-import {commonColors, space, borderRadius} from '@workday/canvas-kit-react/tokens';
-import {hideMouseFocus, GrowthBehavior, generateUniqueId} from '@workday/canvas-kit-react/common';
-
-export interface MenuProps extends GrowthBehavior, React.HTMLAttributes<HTMLUListElement> {
+import {borderRadius, commonColors, space, type} from '@workday/canvas-kit-react/tokens';
+import {
+  createComponent,
+  ElementComponent,
+  ExtractProps,
+  hideMouseFocus,
+  GrowthBehavior,
+  styled,
+  useUniqueId,
+  StyledType,
+} from '@workday/canvas-kit-react/common';
+
+export interface MenuProps extends GrowthBehavior, ExtractProps<typeof Card, never> {
```

Comment:
This changes the interface from extending `React.HTMLAttributes<HTMLUListElement>` to `React.HTMLAttributes<HTMLUListElement>`